contents/handbook/strategy/roadmap.md

```diff
@@ -4,37 +4,43 @@ sidebar: Handbook
 showTitle: true
 ---
 
-Our mission is to increase the number of successful products in the world. 
+Our mission is to 
 
-Our roadmap for 2021 will do three things:
-1. Reach parity with other product analytics tools, but better
-2. Ensure we have the best developer platform for event-based analytics
-3. Make it easy to deploy and maintain a scalable version of PostHog, for free
+> "Increase the number of successful products in the world."
 
-# 1. Reach parity
+Our vision for 2023 is:
 
-PostHog went from 0 to a fully-featured analytics platform in an incredibly short period of time. Realistically, that did mean we had to cut some corners when it came to stability, usability and depth of analytics you could do with PostHog.
+> “Everyone building a product has a clear path to making it successful without losing control of their data.”
 
-In 2021, we're going to make sure that PostHog has all the features you love from other platforms. That way, self-hosting your product analytics doesn't need to be a compromise.
+Our roadmap for the second half of 2021 will do three things:
+1. Diagnosing causes
+2. Collaboration
```

Comment:
I wonder if we should consider experimentation in the mix. It's an intuitive next step to Diagnosing causes, once you know why your users are failing, you run experiments to improve. It's also something that has been asked by users and required for our own dogfooding.

Reply:
Had a long debate about this with @marcushyett-ph and @timgl recently.

We think there will be significant issues with experiments not being statistically significant so shouldn't go ultra deep here. Some very basic UX does seem worth it, highlighting if it's significant or not, but I doubt we'll build a world changing experimentation product.

Whereas ie iteratively will work because it's marketing focussed and at the top of funnel, not deep inside a product where sample sizes are orders of magnitude smaller.

I'd probably focus the way we think about this on "releasing changes safely and understanding their impact" not "doing really statistically significant stuff"

Reply:
The reason I didn't include it was because I thought I'd try and stick to 3 things, that I don't think that's a valid reason not to include it.

So I'll add it in - I think we can be a bit ambiguous about what it will look like and do exactly, but the key motivations are - we want people to be able to release stuff safely and understand the impact of their new features.

A quick plug for my offsite hackathon project - I'd love to work on a super-simple experimentation tool within PostHog that uses P-values to give a rough confidence in how good your results are. Rather than being stat-sig or not when we have 95% confidence - we can be transparent and flexible and say. We are e.g. 75% certain this change was positive...

Reply:
I think that makes. To clarify, 100% aligned that it doesn't have to be a strong requirement on the statistical significance and other more advanced metrics, but things like multivariate support are quite important and part of this "Experimentation" milestone.

I would also argue that our customers could indeed have statistically significant numbers, particularly if you're B2C you generally have larger numbers of volume even if you're early stage.